src/plugins/telemetry/public/components/opt_in_message.tsx

```diff
@@ -9,35 +9,83 @@
 }
 
-export class OptInMessage extends React.PureComponent<Props> {
-  render() {
-    return (
-      <React.Fragment>
-        <FormattedMessage
-          id="telemetry.telemetryBannerDescription"
-          defaultMessage="Want to help us improve the Elastic Stack? Data usage collection is currently disabled. Enabling data usage collection helps us manage and improve our products and services. See our {privacyStatementLink} for more details."
-          values={{
-            privacyStatementLink: (
-              <EuiLink
-                href={this.props.telemetryConstants.getPrivacyStatementUrl()}
-                target="_blank"
-                rel="noopener"
-              >
-                <FormattedMessage
-                  id="telemetry.welcomeBanner.telemetryConfigDetailsDescription.telemetryPrivacyStatementLinkText"
-                  defaultMessage="Privacy Statement"
-                />
-              </EuiLink>
-            ),
-          }}
-        />
-      </React.Fragment>
-    );
+export const OptInMessage: React.FC<OptInMessageProps> = ({
+  addBasePath,
+  telemetryService,
+  telemetryConstants,
+  onClick,
+}) => {
+  return (
+    <React.Fragment>
+      <FormattedMessage
+        id="telemetry.dataManagementDisclaimerPrivacy"
+        defaultMessage="Usage collection (also known as Telemetry) is {optInStatus}.
+          This allows us to learn what our users are most interested in so we can improve our products and services.
+          Refer to our {privacyStatementLink}."
+        values={{
+          optInStatus: telemetryService.isOptedIn ? (
+            <FormattedMessage id="telemetry.enabledStatus" defaultMessage="enabled" />
+          ) : (
+            <FormattedMessage id="telemetry.disabledStatus" defaultMessage="disabled" />
+          ),
+          privacyStatementLink: (
+            /* eslint-disable-next-line @elastic/eui/href-or-on-click */
+            <EuiLink
+              onClick={onClick}
+              href={telemetryConstants.getPrivacyStatementUrl()}
+              target="_blank"
+              rel="noopener"
+            >
+              <FormattedMessage
+                id="telemetry.dataManagementDisclaimerPrivacyLink"
+                defaultMessage="Privacy Statement"
+              />
+            </EuiLink>
+          ),
+        }}
+      />{' '}
+      {renderTelemetryEnabledOrDisabledText(telemetryService, addBasePath, onClick)}
+    </React.Fragment>
+  );
+};
+
+function renderTelemetryEnabledOrDisabledText(
+  telemetryService: TelemetryService,
+  addBasePath: (url: string) => string,
+  onClick?: () => unknown
+) {
+  if (!telemetryService.userCanChangeSettings || !telemetryService.getCanChangeOptInStatus()) {
+    return null;
   }
+
+  const isOptedIn = telemetryService.getIsOptedIn();
+  const actionMessage = isOptedIn ? (
+    <FormattedMessage
+      id="telemetry.dataManagementDisableCollectionLink"
+      defaultMessage="Disable it here."
```

Comment:
This link follows the sentence "Privacy statement" so it sounds like we are asking the user to disable the Privacy statement.  Suggest changing this to:

> Disable usage collection